src/common/components/repositories-home/index.js

```diff
@@ -64,11 +68,13 @@ class RepositoriesHome extends Component {
   }
 
   renderRepositoriesList() {
-    const { snaps, snapBuilds } = this.props;
+    // TODO: bartaz refactor
+    // should it fetch data for first time heading (it already does need it anyway probably)
```

Comment:
FirstTimeHeading needs data about snaps and builds (to display progress) this data is currently fetched by parent component even if it doesn't directly need it.

It even doesn't pass it to FTH, because it uses connect on its own and gets data from selectors...

This is a bit strange case. We have it in 2 places (my repos and select repositories - where first time heading is shown).